<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="shapka" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1901</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="main_button1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>О сотрудниках</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Анализ сети</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус сервера</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button5">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Отчеты</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button4">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Активные БД</string>
     </property>
    </widget>
    <widget class="QWidget" name="logo" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>-1</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button0">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Главная</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="under_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1901</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="buttons_1">
      <item>
       <widget class="QPushButton" name="pushButton_1_1">
        <property name="text">
         <string>Общая статистика</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_1_3">
        <property name="text">
         <string>Просмотр профиля</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_1_2">
        <property name="text">
         <string>Просмотр действий пользователя</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="mainopen1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1901</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="QWidget" name="changes_active1_1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="activeopen1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active1_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active1_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <zorder>activeopen1</zorder>
    <zorder>changes_active1_2</zorder>
    <zorder>changes_active1_3</zorder>
    <zorder>changes_active1_1</zorder>
   </widget>
   <widget class="QWidget" name="under_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1901</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="buttons_2">
      <item>
       <widget class="QPushButton" name="pushButton_2_1">
        <property name="text">
         <string>Просмотр деятельности в сети определенного ПК</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2_2">
        <property name="text">
         <string>Просмотр графа локальной сети</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="under_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1901</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="buttons_3">
      <item>
       <widget class="QPushButton" name="pushButton_3_1">
        <property name="text">
         <string>Информация о сервере</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3_2">
        <property name="text">
         <string>Диспетчер задач</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="under_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1901</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="buttons_4">
      <item>
       <widget class="QPushButton" name="pushButton_4_1">
        <property name="text">
         <string>Просмотр и редактирование БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4_2">
        <property name="text">
         <string>Активные БД на сервере</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="under_5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1901</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="buttons_5">
      <item>
       <widget class="QPushButton" name="pushButton_5_2">
        <property name="text">
         <string>Отчет по сотрудникам</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5_4">
        <property name="text">
         <string>Отчет о использовании сети</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="mainopen2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1901</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="QWidget" name="changes_active2_1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="activeopen2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active2_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
     <widget class="QWidget" name="graph" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>891</width>
        <height>911</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="info_about" native="true">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>0</y>
        <width>521</width>
        <height>911</height>
       </rect>
      </property>
      <widget class="QLabel" name="name_about">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>521</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>О блоке из локальной сети:</string>
       </property>
      </widget>
      <widget class="QWidget" name="network_statistic_widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>521</width>
         <height>671</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="logs_online" native="true">
      <property name="geometry">
       <rect>
        <x>1410</x>
        <y>10</y>
        <width>491</width>
        <height>911</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="about_network" native="true">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>730</y>
        <width>521</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mainopen3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1901</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="QWidget" name="changes_active3_1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="activeopen3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active3_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainopen4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1901</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="QWidget" name="changes_active4_1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="activeopen4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active4_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <zorder>activeopen4</zorder>
    <zorder>changes_active4_2</zorder>
    <zorder>changes_active4_1</zorder>
   </widget>
   <widget class="QWidget" name="mainopen5" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1901</width>
      <height>931</height>
     </rect>
    </property>
    <widget class="Line" name="activeopen5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active5_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="changes_active5_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1901</width>
       <height>911</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainopen0" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1901</width>
      <height>991</height>
     </rect>
    </property>
    <widget class="Line" name="activeopen0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1921</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="main_png" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>1901</width>
       <height>981</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>shapka</zorder>
   <zorder>under_1</zorder>
   <zorder>mainopen1</zorder>
   <zorder>under_2</zorder>
   <zorder>under_3</zorder>
   <zorder>under_4</zorder>
   <zorder>under_5</zorder>
   <zorder>mainopen2</zorder>
   <zorder>mainopen3</zorder>
   <zorder>mainopen5</zorder>
   <zorder>mainopen0</zorder>
   <zorder>mainopen4</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
